test/e2e/multikueue/e2e_test.go

```diff
@@ -477,11 +477,11 @@ var _ = ginkgo.Describe("MultiKueue", func() {
 							Active:    0,
 							Succeeded: 1,
 						},
-						util.IgnoreConditionTimestampsAndObservedGeneration))
```

Comment:
Why this is changed also?

Reply:
It is not need for this object.

Reply:
/hold 
Just to clarify this

Reply:
I will prepare dedicated PR to fix this too.

Reply:
Cool
/hold cancel

Reply:
Done https://github.com/kubernetes-sigs/kueue/pull/3033.